config/environment.js

```diff
@@ -29,30 +29,31 @@ module.exports = function(environment) {
         }]
       ]
     },
-
-    web3ProviderUrl: "https://parity.kosmos.org:8545",
-    ethereumChain: "testnet",
+    contractMetadata: {},
     ipfs: {
       host: 'localhost',
       port: '5001',
       protocol: 'http'
     }
   };
+  if (process.env.KREDITS_CONTRACT_ADDR) {
```

Comment:
yep. I only see those used for configs that must not be in the repo / only for the current machine. 

so defaulting to parity.kosmos.org:8545 and this PR is ok?